src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1296,7 +1296,7 @@
     <value>When an exception is raised in a finally clause, the new exception hides the active exception. This makes the original error difficult to detect and debug.</value>
   </data>
   <data name="DoNotRaiseExceptionsInExceptionClausesMessageFinally" xml:space="preserve">
-    <value>Do not raise an exception from within a finally clause. </value>
+    <value>Do not raise an exception from within a finally clause. .</value>
```

Comment:
Can we fix this string?

Reply:
Crap I reverted this change too... This PR is endless xD